pkg/cloud/powervs.go

```diff
@@ -236,6 +237,7 @@ func (p *powerVSCloud) GetImageByID(imageID string) (*PVMImage, error) {
 func (p *powerVSCloud) CreateDisk(volumeName string, diskOptions *DiskOptions) (disk *Disk, err error) {
 	var volumeType string
 	capacityGiB := util.BytesToGiB(diskOptions.CapacityBytes)
+	diskSize := float64(capacityGiB)
```

Comment:
can this be merged with the above line itself: `float64(util.BytesToGiB(diskOptions.CapacityBytes))`

Reply:
fixed